gcloud/datastore/entity.py

```diff
@@ -106,7 +106,11 @@ def __ne__(self, other):
         :rtype: boolean
         :returns: False if the entities compare equal, else True.
         """
-        return not self == other
+        eq_val = self.__eq__(other)
```

Comment:
@tseaver I'm open to other ideas here (e.g. just returning `False` instead of `NotImplemented` in `__eq__`).